package.json

```diff
@@ -69,19 +72,18 @@
     "@typescript-eslint/eslint-plugin": "^4.17.0",
     "@typescript-eslint/parser": "^4.17.0",
     "auto-changelog": "^2.2.1",
+    "babel-jest": "^27.5.1",
     "codecov": "^3.8.1",
     "eslint": "^7.22.0",
     "eslint-config-prettier": "^8.1.0",
     "eslint-plugin-prettier": "^3.3.1",
     "husky": "^5.2.0",
-    "jest": "^27.4.3",
+    "jest": "^27.5.1",
```

Comment:
I upgraded Jest too because it was emitting warnings after I installed `babel-jest` (some outdated `browserlist` dependencies in v27.4 that have been updated in v27.5)